lib/kitchen/verifier/inspec.rb

```diff
@@ -88,6 +88,8 @@ def call(state)
           v2_loader = ::Inspec::Plugin::V2::Loader.new
           v2_loader.load_all
           v2_loader.exit_on_load_error
+
+          ::Inspec::InputRegistry.instance.cache_inputs = !!config[:cache_inputs] if config[:cache_inputs]
```

Comment:
I'm concerned about tightly coupling the versions of kitchen-inspec and the inspec-core gem. Right now the only constraint in the gemspec is to make sure inspec has plugin support, so it is  a broad range of versions that can be compatible. If we tighten the constraint to be the latest release - presumably a release after inspec/inspec#5211 merges - then people can only use kitchen-inspec with the very latest InSpec.

Could we put a guard clause around this that detects if  Inspec::InputRegistry.instance responds to `cache_inputs=`? That would make it more loosely coupled.

Reply:
Just pushed with a guard, is that what you had in mind?